crates/storage/src/cache.rs

```diff
@@ -1,252 +1,142 @@
-//! Generic cache utility for what we're inserting into the database.
```

Comment:
Can you please re-add the module-level docs. You can change it for whatever you want them to be. I just want to have module-level documentation everywhere possible.